src/taco-cli/cli/remoteBuild/remoteBuildClientHelper.ts

```diff
@@ -81,6 +82,15 @@ class RemoteBuildClientHelper {
                     }
                 }
 
+                // Clear build log if it exists
+                if (fs.existsSync(buildLog)) {
```

Comment:
Unrelated, but now that we have a few filesystem operations going on here, maybe we should stop them being synchronous.